builder/internals.go

```diff
@@ -475,6 +475,7 @@ func calcCopyInfo(b *Builder, cmdName string, cInfos *[]*copyInfo, origPath stri
 	return nil
 }
 
+// ContainsWildcards checks if a string contains any wildcards
 func ContainsWildcards(name string) bool {
```

Comment:
Pretty sure this should be private. Let's fix this. Then you don't need docstring at all :)